python_terraform/terraform.py

```diff
@@ -456,6 +456,13 @@ def show_workspace(self, **kwargs) -> CommandOutput:
         """
         return self.cmd("workspace", "show", **kwargs)
 
+    def list_workspace(self) -> List[str]:
+        """List of workspaces
+        
+        :return: workspaces
+        """
+        return list(filter(lambda x: len(x) > 0, map(lambda x: x.strip('*').strip(), self.cmd("workspace", "list")[1].split())))
```

Comment:
please change format to pylint compliant.
Could you also give an example what would output of workspace looks like?

I'm worried about potential "KeyError" or "indexError" from your hard coded `1` here

Reply:
- used pycodestyle as pylint
- self.cmd returns will always be size 3 but 2nd one can be None. fixed that with checking, so no more KeyError or any other error.